pkg/vpcmodel/multiExplainability.go

```diff
@@ -88,6 +88,8 @@ func (c *VPCConfig) getNodesFromEndpoint(endpoint EndpointElem) ([]Node, error)
 	switch n := endpoint.(type) {
```

Comment:
Somehow this file is not very familiar to me... Lets take it offline.

Reply:
done